feat: new doc, style fixes, wording fixes, link fixes (#79)
publicsq/developers.publicsquare.com: docs/guides/merchants/process-ach-payments.mdx

import Tabs from "@theme/Tabs";
import TabItem from "@theme/TabItem";

import { Alert } from "@site/src/components/shared/Alert";
import GettingStartedSection from "@site/src/components/docs/_getting-started-section.mdx";

# Process ACH Payments

Now that you have completed the [Collect Bank Accounts](/guides/merchants/collect-bank-accounts) guide, let's process an ACH payment with the newly captured bank account.

## Getting Started

<GettingStartedSection />

### Get your Secret Key

Next you will need your `Secret Key`. [Go to your Developers section](https://portal.publicsquare.com/developers/api-keys) and click `Reveal` for your `Secret Key` and copy the value.

<Alert>Save the Secret Key as it will be used in the next steps of this guide.</Alert>

## Process the Payment

> We will need to use the information from the previously collected card to create a new payment.

Reviewer: ```suggestion
We will need to use the information from the previously collected bank account to create a new payment.
```

> For example, given you have a previously collected card:

Reviewer: ```suggestion
For example, given you have a previously collected a bank account:
```

> ```json showLineNumbers title=card.json

Reviewer: ```suggestion
```json showLineNumbers title=bankAccount.json
```

-H 'IDEMPONTENCY-KEY: '09ec2c87-7fb8-44ca-bb18-5c71a76974da' \
-H 'Content-Type: application/json' \
-d '{
  "amount": 1000,
  "currency": "USD",
  "paymentMethod": {
    "bank_account": "ba_7Ay5mcUXAxwrN6wQEQUVEHBCJ"
  },
  "billing_details": {
    "address_line_1": "111 Test St.",
    "city": "Des Moines",
    "state": "IA",
    "postal_code": "51111",
    "country": "US"
  }
}'
```

If you supplied the `billing_details` or [updated the bank account](/api/financial/update-bank-account), then you can omit the `billing_details` section when processing the payment:

```shell showLineNumbers title="Create a Payment with Customer"
curl 'https://api.publicsquare.com/payments' \
-X 'POST' \
-H 'X-API-KEY: <API_KEY_SECRET>' \
-H 'IDEMPONTENCY-KEY: '09ec2c87-7fb8-44ca-bb18-5c71a76974da' \
-H 'Content-Type: application/json' \
-d '{
  "amount": 1000,
  "currency": "USD",
  "paymentMethod": {
    "bank_account": "ba_7Ay5mcUXAxwrN6wQEQUVEHBCJ"
  }
}'
```

Given the above examples, you should see a payment result similar to:

```json showLineNumbers
{
  "id": "pmt_2YKewBonG4tgk12MheY3PiHDy",
  "account_id": "acc_B518niGwGYKzig6vtrRVZGGGV",
  "environment": "test",

>   "status": "successful",

Reviewer: ```suggestion
  "status": "pending",
```

  "transaction_id": "trx_95rvMJvAVeG68W4NtLdfkN3LG",
  "amount": "1000",
  "amount_charged": "1000",
  "amount_refunded": "0",
  "refunded": "false",
  "currency": "USD",
  "payment_method": {
    "bank_account": {
      "id": "ba_7Ay5mcUXAxwrN6wQEQUVEHBCJ",
      "account_holder_name": "John Doe",
      "account_holder_type": "individual",
      "account_type": "individual",
      "routing_number": "123456789",
      "account_number_last4": "1011"
    },

>     "account_balance": {
>       "account_id": "acc_B518niGwGYKzig6vtrRVZGGGV",
>       "account_name": "Test Company, LLC"
>     }

Reviewer: ```suggestion
```
